scintillometry/channelize/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Licensed under the GPLv3 - see LICENSE
+"""Channelization modules."""
+
+from .base import ChannelizeTask
```

Comment:
Refactoring into a single `channelize.py` task is kept separate?

Reply:
My plan is to have this go through, then refactor the entire code as we discussed.